lib/token.js

```diff
@@ -0,0 +1,24 @@
+// Token Generater file for REPL
+
+var crypto;
+var token;
+
+module.exports.generateToken = function () {
+
+	try {
+		crypto = require('crypto')
+		var secret = require('../config/config').secret
+
+		var timeNow = new Date().getTime()
+		var hash = crypto.createHmac('sha256', secret)
+									.update(timeNow + '')
+									.digest('base64')
+		token = {msg_mac : hash, time_created : timeNow}
+	} catch(err) {
+		console.log('crypto support is disabled')
+	}
+
```

Comment:
didn't know about that, [read this up](http://stackoverflow.com/questions/19727905/in-javascript-is-it-expensive-to-use-try-catch-blocks-even-if-an-exception-is-n). In our testing module, do we need to focus this much on performance?